README.md

```diff
@@ -1 +1,123 @@
-# guru-discord-bot
\ No newline at end of file
+<h1 align="center">Guru Discord Bot</h1>
+<div align="center">
+    <strong>
+        Szerveren lévő jogosultságok/skillek kezelő botja a <a href="https://discord.gg/A5yC5Hhxrs">Step into Community</a> Discord szerveren.
+    </strong>
+</div>
+<br/>
+<div align="center">
+
+[![Discord](https://img.shields.io/discord/848942109191962665.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/A5yC5Hhxrs)
+
+
+</div>
+<br/>
+
+## Funkcionalitás
+
+- Csatlakozó felhasználók üdvözlése, ami után reakciók segítségével tudják magukhoz rendelni a szerepköröket
+- A bot által nyújtott parancsok/súgó lekérése (`/skills`)
+- Szerepkörök/skillek listázása a szerveren (`/list-skills`)
+- Szerepkörök felhaszálóhoz rendelése (`/i-know`)
```

Comment:
In the application.properties file, there is a wider description for the command, inluding the attribute, which can be a list of skills too. Can you change this to be similar?

Reply:
Of course.

Reply:
@BeardOfDoom I just took on a look on this comment, and I do not really understand it now.

Reply:
I think @BeardOfDoom meant that the `application.properties` file lists commands as

~~~~
i-know <skill_neve>
~~~~

which indeed includes the parameters, while the README does not.